### DISCUSSION TITLE
Warning if vorta does not start / works properly

### DISCUSSION THREAD

**Discussion by nomis5000** [borgbase/vorta#1746] [General]
Hey, I just noticed that vorta installed via the arch user repository has not been able to start at boot for almost a month and so backups are missing for that period as well. Is there any way to check the proper running of vorta from time to time and display a warning if necessary? I can imagine that someone has already written a reliable script for such a case.

**Comment by real-yfprojects** [comment]
Sorry, you might actually be the first one. Feel free to post/link your solution here when you got one.

**Comment by sten0** [comment]
Hi @nomis5000, Is this a duplicate of #1756, or if not, how is it different?

**Reply by nomis5000** [reply]
It's not that. My problem is that vorta won't start. #1756 is a problem when vorta is already running. I think I will write myself a little notification in my i3 config to remind me to check that vorta has started.

**Reply by real-yfprojects** [reply]
> My problem is that vorta won't start.

Have you looked into the source of that issue? This could be a bug that needs fixing.

**Reply by nomis5000** [reply]
There may have been a problem with a python package update a few months ago, but a reinstall as described in the comments section of the arch/aur package solved it. Since then everything is fine. Next time, I just want to notice the problem earlier. I don't have enough time at the moment, but I will come up with a useful script one day.

**Comment by sten0** [comment]
>>> My problem is that vorta won't start.

@nomis5000 thank you for confirming as well as explaining how your case
is different.

>> Have you looked into the source of that issue? This could be a bug
>> that needs fixing.

@yfprojects I think the nature of this issue is that the OP is
requesting managed execution.  For example, if "systemd --user" was
managing a hypothetical vorta backend daemon, then
https://github.com/rfjakob/systembus-notify could use dbus to notify the
user in case of error.  I'm sorry that I'm not aware of a generic
desktop application execution manager, and it seems the OP isn't either,
hence:

> but I will come up with a useful script one day.

A wrapper script to catch and handle potential errors and notify the
user.  This is the very important systems integration and real-world
user experience scope.  I confess that I have been having to monitor
backup policy health by monitoring the borg repo dir on the backup
server, and have no idea how normal desktop users would ever know that
their backups are silently failing (ie: not happening).
